src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -26,21 +26,25 @@
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --proxy-https https://proxy-url --proxy-http http://proxy-url --proxy-skip-range excludedIP,excludedCIDR,exampleCIDRfollowed,10.0.0.0/24
     - name: Onboard a connected kubernetes cluster by specifying the https proxy, http proxy, no proxy  with cert settings.
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --proxy-cert /path/to/crt --proxy-https https://proxy-url --proxy-http http://proxy-url --proxy-skip-range excludedIP,excludedCIDR,exampleCIDRfollowed,10.0.0.0/24
+    - name: Onboard a connected kubernetes cluster with private link feature enabled by specifying private link parameters.
+      text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --enable-private-link true --private-link-scope-resource-id pls/resource/arm/id
```

Comment:
Do we expect any additional argument for private link in future? Should scope resource itself implies cx wants to enable private link?

Reply:
We do not expect any additional private link argument in future (except when we do the optimization for proxy in which case we would need a static ip from the user).
Keeping only the argument '--private-link-scope-resource-id' might confuse the users when they want to disable private link. It's easy to disable by passing '--enable-private-link false' instead of passing null value for '--private-link-scope-resource-id'. In fact, for patch scenario for disabling private link, the former method could only work. the latter would mean no-op